#!/usr/bin/env python
PACKAGE = "kuka_iiwa_control_hbp"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("dmp_TAU_0", double_t, 0, "DMP's time constant",    1.0, -10.0,   1000.0)
gen.add("dmp_ALPHA_X", double_t, 0, "DMP's spring-damping constant",    4.0, -10.0,   1000.0)
gen.add("dmp_ALPHA_S", double_t, 0, "DMP's phase evolution's constant",    4.0, -10.0,   1000.0)
gen.add("od_K", double_t, 0, "Orientation dynamics gain",    3.0, -10.0,   1000.0)
gen.add("od_fc", double_t, 0, "Orientation dynamics angular speed's filter low-pass cut-off frequency",    5.0, -10.0,   1000.0)

exit(gen.generate(PACKAGE, "kuka_iiwa_control_hbp", "TrajectoryDynamics"))